.eslintrc.json

```diff
@@ -0,0 +1,47 @@
+{
+  "root": true,
+  "ignorePatterns": [
+    "projects/**/*"
+  ],
+  "overrides": [
+    {
+      "files": [
+        "*.ts"
+      ],
+      "extends": [
+        "eslint:recommended",
```

Comment:
you do not have a linter configured for Airbnb rules
write in terminal `ng add @angular-eslint/schematics@latest`
then `npm i -D eslint-config-airbnb-base eslint-plugin-import eslint-config-airbnb-typescript`
add `"airbnb-base"` and after him `"airbnb-typescript/base"` in to `.eslintrc.json` file extends array
at the beginning of this file add `"parserOptions": { "project": ["tsconfig(.app|.spec|).json"] },`
after this use command `ng lint` in terminal to see all errors and warnings
then `ng lint --fix` to fix what can automatically
after this there should be errors "Prefer default export on a file with single export" this can be fixed by turning off this rule in the settings below by adding the line `"import/prefer-default-export": "off"`
and there will be a warning not to use the console, this can be fixed by adding next to it `// eslint-disable-line`